### PR TITLE
Add gatk4/applybqsr

### LINKED ISSUE #196: new module: gatk4/applybqsr
I think it would be good to have module for [gatk4](https://bioconda.github.io/recipes/gatk4/README.html)

### PR DESCRIPTION
**Pull request by yocra3**
## PR checklist

Closes #196 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `<SOFTWARE>.version.txt` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Add a resource `label`
- [X] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [X] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

**Review by KevinMenden — COMMENTED** [review]
This looks great, thanks!

But you'll need to add an entry to the `pytest_software.yml` file in order to get the tests running.

**Comment by yocra3** [comment]
Cool! I have added the code for testing my module to the pytest file and now the tests are running. I have another PR where I added a new module (#327) but I did not modify the pytest file either. I should wait until this PR is merged and then I add the test to pytest, right?

**Review by KevinMenden — APPROVED** [review]

**Pull request merged**
